docs/operate/customize/file-uploads/upload_limits.md

```diff
@@ -28,7 +28,7 @@ APP_RDM_DEPOSIT_FORM_QUOTA = {
 
 While the above mentioned configuration would already prevent the backend from accepting files that are too large, an additional layer of defense can be added by configuring `nginx` to reject client requests above a certain size.
 
-This can be achieved by setting the `client_max_size_body` for the REST API file content endpoint (`location ~ /api/records/.+/draft/files/.+/content`) to a desired value, e.g. `75G`.
+This can be achieved by setting the `client_max_size_body` for the REST API file content endpoint (`location ~ /api/records/.+/draft/files/.+/content(/[0-9]+)?/?$`) to a desired value, e.g. `75G`.
```

Comment:
Interesting, the multipart upload takes a slightly different URL? The regex will account for regular upload too so it should be fine.